editor/src/components/inspector/common/css-utils.ts

```diff
@@ -623,13 +642,13 @@ export function gridCSSKeyword(
   }
 }
 
-export function gridCSSRepeat(times: number, value: GridDimension[]): GridCSSRepeat {
+export function gridCSSRepeat(times: GridCSSRepeatTimes, value: GridDimension[]): GridCSSRepeat {
   return {
     type: 'REPEAT',
     times: times,
     value: value,
     areaName: null,
-  }
+  } as GridCSSRepeat
```

Comment:
Unfortunately it is required because the `times` assignment is not liked by TS. I'd be open to better solutions actually (it might mean reworking the types definition tho).